src/NuGet/Microsoft.Net.CSharp.Interactive.netcore.nuspec

```diff
@@ -17,7 +17,7 @@
     <projectUrl>$projectUrl$</projectUrl>
     <tags>$tags$</tags>
     <dependencies>
-      <group targetFramework="NETCoreApp1.0">
+      <group targetFramework="netcoreapp1.1">
```

Comment:
What verificatioin do we have that thius is correct?

Reply:
Manual right now. I went and checked each assembly/file included in the package (and its dependent packages).

CsiCore is currently targeting netcoreapp1.1: https://github.com/dotnet/roslyn/blob/master/src/Interactive/CsiCore/CsiCore.csproj#L13

And for the other package, CscCore and VbcCore are currently targeting netcoreapp2.0: https://github.com/dotnet/roslyn/blob/master/src/Compilers/CSharp/CscCore/CscCore.csproj#L15 and https://github.com/dotnet/roslyn/blob/master/src/Compilers/VisualBasic/VbcCore/VbcCore.csproj#L15

Reply:
I also validated the deps.json and the runtimeconfig.json in the latest signed build output for these projects